src/include/units/chrono.h

```diff
@@ -36,11 +37,15 @@ struct quantity_like_traits<std::chrono::duration<Rep, Period>> {
   [[nodiscard]] static constexpr rep count(const std::chrono::duration<Rep, Period>& q) { return q.count(); }
 };
 
+template <typename C>
+struct chrono_clock_point_origin : point_origin<chrono_clock_point_origin<C>, typename dimension_unit<physical::si::dim_time>::reference> {};
+
 template<typename C, typename Rep, typename Period>
 struct quantity_point_like_traits<std::chrono::time_point<C, std::chrono::duration<Rep, Period>>> {
   using dimension = physical::si::dim_time;
   using unit = downcast_unit<dimension, ratio(Period::num, Period::den)>;
   using rep = Rep;
+  using origin = chrono_clock_point_origin<typename std::chrono::time_point<C, std::chrono::duration<Rep, Period>>::clock>;
```

Comment:
```suggestion
  using origin = chrono_clock_point_origin<C>;
```

Reply:
I wasn't quite sure if that is guaranteed by the standard. But probably any program where it isn't is invalid.

Reply:
Yes, it's guaranteed: http://eel.is/c++draft/time#point.